src/connection/index.js

```diff
@@ -208,23 +211,20 @@ class ConnectionFSM extends BaseConnection {
           ))
         }
 
-        if (circuitTried) {
-          return this.close(Errors.CONNECTION_FAILED(
-            new Error(`No available transports to dial peer ${this.theirB58Id}!`)
-          ))
-        }
+        return this.close(Errors.CONNECTION_FAILED(
+          new Error(`No available transports to dial peer ${this.theirB58Id}!`)
```

Comment:
add `errcode` here?

Reply:
Ok, my bad. So, we do not need to create the `new Error` here. It can be omitted and `errCode` will handle it.